web/modules/custom/server_general/src/ThemeTrait/ElementWrapThemeTrait.php

```diff
@@ -80,11 +81,13 @@ protected function wrapContainerNarrow(array $element, BackgroundColorEnum $bg_c
    *   Render array.
    * @param \Drupal\server_general\ThemeTrait\Enum\AlignmentEnum $align
    *   Determine the alignment of flex.
+   * @param bool $full_width
+   *   Whether the elements should occupy full width of parent.
    *
    * @return array
    *   Render array.
    */
-  protected function wrapContainerVerticalSpacing(array $element, AlignmentEnum $align = AlignmentEnum::Default): array {
+  protected function wrapContainerVerticalSpacing(array $element, AlignmentEnum $align = AlignmentEnum::Default, bool $full_width = FALSE): array {
```

Comment:
Why is `full_width` required for vertical spacing? Maybe it needs a new wrapper, that does a single thing?